htdocs/class/logger/filelogger.php

```
@@ -0,0 +1,533 @@
+        if (false === $size || ($size + $incoming) < $this->maxSize) {
+            return;
+        }
+
+        @unlink($this->file . '.' . $this->maxFiles);
+        for ($i = $this->maxFiles - 1; $i >= 1; --$i) {
+            if (is_file($this->file . '.' . $i)) {
+                @rename($this->file . '.' . $i, $this->file . '.' . ($i + 1));
+            }
+        }
+        @rename($this->file, $this->file . '.1');
+    }
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 518. (undefined)

(ErrorControlOperator)